src/references/NCBITaxon/NCBITaxon_12species.owl.xml.rdf

```diff
@@ -1,190 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
In the description, you said that you were renaming these two files, but it looks like they've been deleted here. Is it possible that the new file wasn't added in your commit? Or maybe the description is just out of date?

Reply:
Rather than "renaming", I deleted and created new files.  So you should see 2 deleted files and 2 added files with somewhat similar names.  Does that seem correct?

Reply:
Hmmm... I don't see the new files added here, but perhaps Github isn't updating properly?

Reply:
![image](https://user-images.githubusercontent.com/23193756/77008116-f0133900-693b-11ea-8b23-35b44d1a744d.png)